polaris-react/src/components/Collapsible/Collapsible.tsx

```diff
@@ -51,11 +53,14 @@ export function Collapsible({
     expandOnPrint && styles.expandOnPrint,
   );
 
+  const transitionDisabled = transition === false;
```

Comment:
I think we still care about transition.duration being `0 || 0ms || 0s` here. Can we disable transition on those values as well?